### ISSUE TITLE
Accessibility: MAS36: Keyboard: Carousel Buttons are not accessible with keyboard.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#38]
#A11yMAS
#MAS36
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1.Open [https://code.visualstudio.com/docs ](https://code.visualstudio.com/docs )in edge.
2.Navigate to "Visual Studio Code in Action" below the header of the page .
3.Try to access the carousel buttons below the image .[screenshot 1]
**_Note: Same issue is observed for below link:_**
[https://code.visualstudio.com/](https://code.visualstudio.com/) [refer screenshot 2]

**Actual Result:**
Unable to access the "Carousel" buttons below the image while navigating with keyboard.

**Expected Result:**
The user should be able  to access the all the interactive elements available in the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may find it difficult while navigating with the tab to use the Application. 

**Screenshots:**
![1_crousels are not accessible with keyboard](https://cloud.githubusercontent.com/assets/24825441/21712846/e1d80e9a-d41c-11e6-80a9-e6cff8528b4f.png)
![2_crousels are not accessible with keyboard](https://cloud.githubusercontent.com/assets/24825441/21712847/e1d868fe-d41c-11e6-9905-ba9a1432cd3d.png)

**Comment by chrisdias** [comment]
You can set focus to the image in the carousel and press right/left arrow.
